src/app/components/home/home.component.ts

```diff
@@ -22,6 +24,8 @@ export class HomeComponent extends SubscriptionsComponent implements OnInit, Aft
 	moduleHeight: number;
 	resizeSensor: ResizeSensor;
 
+	faTwitter = faTwitter;
```

Comment:
I don't see this used explicitly in the template, is it necessary to be imported in the component?